SeQuant/core/abstract_tensor.hpp

```diff
@@ -349,7 +352,7 @@ class AbstractTensor {
 /// objects.
 /// @{
 inline auto braket(const AbstractTensor& t) { return t._braket(); }
-inline auto indices(const AbstractTensor& t) { return t._braketaux(); }
+inline auto braketaux(const AbstractTensor& t) { return t._braketaux(); }
```

Comment:
Why the rename? I much prefer having a function which will give me _all_ indices instead of a function that gives me indices in the given groups (which can't be amended in case new groups get added and hence code relying on getting all indices needs to be refactored - but one has to explicitly think of that

Reply:
renamed for consistency and explicitness, the meaning is the same. We had `braket()` already. We can keep both, if you have a lot of code that uses `indices()` already.

the bigger issue is that the distinction between slots and indices is not explicit in these functions ...

Reply:
> I disagree in a general context. The meaning is the same _now_ because we don't have more index groups but as soon as we add another group, the meaning will no longer be equivalent. Exactly as happened to `braket`.

I sure hope we don't :)

Reply:
> I'd definitely appreciate keeping the `indices` function, albeit for semantic reasons rather than having a lot of code that uses it.

done